### PR TITLE
Feature/androidx

### PR DESCRIPTION
**Pull request by jakubjodelka**

### CODE REVIEW

**Comment by jakubjodelka** [comment]
fixes for #6 #8 #11 issues

**Comment by jakubjodelka** [comment]
and also #9 after deleting old distribution way

**Review by lukkob — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -1,17 +1,16 @@
 buildscript {
-    ext.kotlin_version = '1.2.51'
-    ext.support_version = '27.1.1'
-    ext.constraint_version = '1.1.2'
+    ext.kotlin_version = '1.3.50'
```
Comment (DawidJamrozy) [inline build.gradle]: maybe update kotlin to 1.3.61?

File: build.gradle [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+    ext.androidx_version = "1.1.0"
```
Comment (DawidJamrozy) [inline build.gradle]: duplicate in build.gradle line 36

File: fluidbottomnavigation/script/version.gradle [hunk 3]
```diff
@@ -1,12 +1,8 @@
...
 ext {
-    getLibraryVersionCode = { ->
-        return (System.getenv("BITRISE_BUILD_NUMBER") ?: "0").toInteger()
-    }
     getLibraryVersionName = { ->
-        return getLibraryMinorVersion() + "." + getLibraryVersionCode().toString()
+        return "1.2"
+    }
+    getLibraryVersionCode = { ->
+        return 120
```
Comment (DawidJamrozy) [inline fluidbottomnavigation/script/version.gradle]: is it safe to leave hard coded version number here?

**Review by DawidJamrozy — COMMENTED** [review]
Look at my comments, nothing major

File: build.gradle (hunk 2 above)
Comment (jakubjodelka) [reply build.gradle]: fixed

File: build.gradle (hunk 1 above)
Comment (jakubjodelka) [reply build.gradle]: updated

**Pull request merged**